src/pages/updateTitle.vue

```diff
@@ -0,0 +1,39 @@
+<template>
+    <span class=" text-7xl flex justify-center " v-if="isLoading">
```

Comment:
this page not needed, this all functionality will be inside the poll update page @ShivanshSinghnnovative